jupyter-js-widgets example fails to install

The readme [here](https://github.com/ipython/ipywidgets/blob/master/jupyter-js-widgets/examples/web/README.md) states to first do `npm install` inside `ipywidgets/ipywidgets`:

```bash
(py35)
klay6683 at macd2860 in ~/src/ipywidgets/ipywidgets (master)
$ npm install
npm WARN enoent ENOENT: no such file or directory, open '/Users/klay6683/src/ipywidgets/ipywidgets/package.json'
npm WARN ipywidgets No description
npm WARN ipywidgets No repository field.
npm WARN ipywidgets No README data
npm WARN ipywidgets No license field.
(py35)
```
Don't know if that is okay. I then went ahead, following the next instruction to go into `this` directory, where the readme is located, so in this case, `ipywidgets/jupyter-js-widgets/examples/web` and executed another `npm install` for which the outcome looks unsuccessful:

```bash
klay6683 at macd2860 in ~/src/ipywidgets/jupyter-js-widgets/examples/web (master)
$ npm install

> jupyter-js-widgets@0.0.16 prepublish /Users/klay6683/src/ipywidgets/jupyter-js-widgets
> npm run build


> jupyter-js-widgets@0.0.16 build /Users/klay6683/src/ipywidgets/jupyter-js-widgets
> npm run css


> jupyter-js-widgets@0.0.16 precss /Users/klay6683/src/ipywidgets/jupyter-js-widgets
> rimraf ./css/

sh: rimraf: command not found

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/5.7.1/bin/node" "/usr/local/bin/npm" "run" "css"
npm ERR! node v5.7.1
npm ERR! npm  v3.6.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! jupyter-js-widgets@0.0.16 precss: `rimraf ./css/`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the jupyter-js-widgets@0.0.16 precss script 'rimraf ./css/'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyter-js-widgets package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rimraf ./css/
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyter-js-widgets
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyter-js-widgets
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/klay6683/src/ipywidgets/jupyter-js-widgets/npm-debug.log

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/5.7.1/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v5.7.1
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! jupyter-js-widgets@0.0.16 build: `npm run css`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyter-js-widgets@0.0.16 build script 'npm run css'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyter-js-widgets package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run css
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyter-js-widgets
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyter-js-widgets
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/klay6683/src/ipywidgets/jupyter-js-widgets/npm-debug.log

npm ERR! addLocal Could not install /Users/klay6683/src/ipywidgets/jupyter-js-widgets
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/5.7.1/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.7.1
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! jupyter-js-widgets@0.0.16 prepublish: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyter-js-widgets@0.0.16 prepublish script 'npm run build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyter-js-widgets package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyter-js-widgets
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyter-js-widgets
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/klay6683/src/ipywidgets/jupyter-js-widgets/examples/web/npm-debug.log
(py35)
```
I tried this on github master today. `npm` is installed via `homebrew`, version 5.7.1.

--- a/jupyter-js-widgets/examples/web/README.md
+++ b/jupyter-js-widgets/examples/web/README.md
@@ -4,12 +4,12 @@ This directory is an example project that shows how you can embed the widgets in
 a context other than the notebook.
 
 ## Try it
-1. Open a command line in the `ipywidgets/ipywidgets` subdirectory and run `npm install`.
+1. Start with a development install of jupyter-js-widgets by running `npm install` in the `jupyter-js-widgets` subfolder of the repo root (see the [README.md](../../../README.md) in the repo root for more details).
 2. Cd into this directory and run `npm install`.
 3. Now open the `index.html` file.
 
 ## Details
-If you plan to reproduce this in your own project, pay careful attention to the 
+If you plan to reproduce this in your own project, pay careful attention to the
 package.json file.  The dependency to jupyter-js-widgets, which reads
 `"jupyter-js-widgets": "file:../../../ipywidgets"`, **should not** point to `"file:../../../ipywidgets"`.
 Instead point it to the version you want to use on npm.

--- a/jupyter-js-widgets/examples/web2/README.md
+++ b/jupyter-js-widgets/examples/web2/README.md
@@ -4,12 +4,12 @@ This directory is an example project that shows how you can embed the widgets in
 a context other than the notebook.
 
 ## Try it
-1. Open a command line in the `ipywidgets/ipywidgets` subdirectory and run `npm install`.
+1. Start with a development install of jupyter-js-widgets by running `npm install` in the `jupyter-js-widgets` subfolder of the repo root (see the [README.md](../../../README.md) in the repo root for more details).
 2. Cd into this directory and run `npm install`.
 3. Now open the `index.html` file.
 
 ## Details
-If you plan to reproduce this in your own project, pay careful attention to the 
+If you plan to reproduce this in your own project, pay careful attention to the
 package.json file.  The dependency to jupyter-js-widgets, which reads
 `"jupyter-js-widgets": "file:../../../ipywidgets"`, **should not** point to `"file:../../../ipywidgets"`.
 Instead point it to the version you want to use on npm.

--- a/jupyter-js-widgets/examples/web3/README.md
+++ b/jupyter-js-widgets/examples/web3/README.md
@@ -4,14 +4,14 @@ This directory is an example project that shows how you can embed the widgets in
 a context other than the notebook.
 
 ## Try it
-1. Open a command line in the `ipywidgets/ipywidgets` subdirectory and run `npm install`.
+1. Start with a development install of jupyter-js-widgets by running `npm install` in the `jupyter-js-widgets` subfolder of the repo root (see the [README.md](../../../README.md) in the repo root for more details).
 2. Cd into this directory and run `npm install`.
 3. Run `npm run host`
 4. In a new terminal run `python -m notebook --no-browser --NotebookApp.allow_origin="*"`
 5. In a web browser, navigate to `http://localhost:8080/` (or the address specified by the `npm run host` command)
 
 ## Details
-If you plan to reproduce this in your own project, pay careful attention to the 
+If you plan to reproduce this in your own project, pay careful attention to the
 package.json file.  The dependency to jupyter-js-widgets, which reads
 `"jupyter-js-widgets": "file:../../../ipywidgets"`, **should not** point to `"file:../../../ipywidgets"`.
 Instead point it to the version you want to use on npm.

